tools/commands.js

```diff
@@ -486,7 +486,21 @@ main.registerCommand({
     utils.validatePackageNameOrExit(
       packageName, {detailedColonExplanation: true});
 
-    var fsName = colonConverter.convert(packageName);
+    // When we create a package, avoid introducing a colon into the file system
+    // by naming the directory after the package name without the prefix.
+    var fsName = packageName;
+    if (packageName.indexOf(":") !== -1) {
+      var split = packageName.split(":");
+
+      if (split.length > 2) {
+        Console.error(packageName +
```

Comment:
Yeah, I think you're probably right. When I talked to @debergalis he said we should disallow it, but now I'm maybe edging towards just using the last part? I'm not really sure what the best option here is, but in the future we should probably validate everything more strictly when we introduce names like this.